packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -186,7 +186,8 @@ contract L1StandardBridge_Receive_Test is CommonTest {
         vm.prank(alice, alice);
         (bool success,) = address(l1StandardBridge).call{ value: 100 }(hex"");
         assertEq(success, true);
-        assertEq(address(optimismPortal).balance, 100);
+        assertEq(address(optimismPortal).balance, 0);
+        assertEq(address(sharedLockbox).balance, 100);
```

Comment:
Missing  `assertEq(address(sharedLockbox).balance, 0);` at the beggining of the test as well

Reply:
I will add the check at the beginning, but for now will leave the portal one also

Reply:
Perfect